### ISSUE TITLE
Different values, using P6_1 till P6_6 (chicken feet)

### ISSUE THREAD

**Issue by kurtreit** [opentx/opentx#8898]
It is not possible to create logical switches, using the chicken feet (P6), because companion generates different outputs for P6_1 till P6_6.
Using companion 2.3.15_N658
For instance P6_1 returns values from -54% till -60%

**Comment by jrwieland** [comment]
I just checked that nightly (Because I do use the "chicken feet" for my flight modes). 

Initially the values were off when I tried to set them.

After rolling the 6 position back and forth a couple of times , the settings the values adjusted out to what I would expect them to be .

here is a screenshot of the values they worked at LS1 = 6p1
![companion](https://user-images.githubusercontent.com/20526716/162195105-593bcf97-314a-4a14-9ce7-ecb561a8c67d.JPG)

**Comment by kurtreit** [comment]
I used the radio profile Jumper T18. There is no chance to get predictable results. P6 is assigned to Channel 10.
![image](https://user-images.githubusercontent.com/97182580/162205041-7dd49175-ecf7-401d-a05b-47dd7f3026dc.png)
Should be -60%
![image](https://user-images.githubusercontent.com/97182580/162205279-58344fef-6e77-4c35-ace8-199e77b27aed.png)
Should be -100%

**Comment by bigtwin** [comment]
I can confirm. 2.3.14/Horus on MAC OSX. The values for each position of 6P differ heavily in _Companion_. I noticed that the value for the switch seems to be "triggered" my the mouse movement/position rather than by the actual switch position indicator (the little, hard-to-tell triangle).

**Comment by jrwieland** [comment]
I do agree that using a mouse click does yield unpredictable results as apposed to using the wheel on the mouse to manipulate the 6p switch the values; using a click varied as much as 6% [-94 vs. -100]  (from my quick check) using the wheel of the mouse they stay constant (-100 to 100).

Initially, I wasn't able to replicate what was stated in the issue, and since I use them from my flight modes, this was a great concern worth trying for myself.  (I did create a t18 radio and tested it after the comment) and using the mouse to click the 6p to a new position did result in unpredictable values as bigtwin stated.

**Comment by kurtreit** [comment]
@bigtwin 
Yes, you are in right! I think they should rework the snapping position of the mouse cursor for exact assignment.

**Comment by kurtreit** [comment]
@jrwieland 
With the wheel, its fine
